File: api/src/main/java/net/onelitefeather/pandorascluster/api/util/DefaultPlayerImpl.java
package net.onelitefeather.pandorascluster.api.util;

import net.onelitefeather.pandorascluster.api.enums.Permission;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

import java.util.UUID;

public final class DefaultPlayerImpl implements PlayerUtil {
    @Override
    public @Nullable Object getPlayer(@NotNull UUID var1) {
        throw new IllegalStateException("This is not implemented yet!");
    }

    @Override
    public boolean isOnline(@NotNull UUID var1) {
        return false;
    }

    @Override
    public boolean hasPermission(@NotNull UUID var1, @NotNull String var2) {
        return false;
    }

    @Override
    public boolean hasPermission(@NotNull UUID var1, @NotNull Permission var2) {
        return false;
    }
}

File: api/src/main/kotlin/net/onelitefeather/pandorascluster/api/utils/DefaultPlayerImpl.kt
package net.onelitefeather.pandorascluster.api.utils

import jdk.jshell.spi.ExecutionControl.NotImplementedException
import java.util.*

class DefaultPlayerImpl : PlayerUtil {
    override fun getPlayer(uuid: UUID): Any {
        throw NotImplementedException("This is not implemented yet!")
    }

    override fun isOnline(uuid: UUID) = false

    override fun hasPermission(uuid: UUID, permission: String) = false
}